hw/ip/pattgen/data/pattgen.prj.hjson

```diff
@@ -11,7 +11,7 @@
       {
         version:            "1.0",
         life_stage:         "L1",
-        design_stage:       "D1",
+        design_stage:       "D2",
         verification_stage: "V1",
         dif_stage:          "S0",
```

Comment:
@cfrantz @mateenfaisal 
Who is the SW owner of this DIF?
It would be good if someone from the SW side could have a look at the CSRs before we move this into D2...

Reply:
afaik, we don't have a SW owner for PATTGEN DIF but someone from SW side must look into it. @asb/@cfrantz   - can you please allocate a person from SW team to review PATTGEN CSR?

Reply:
@mcy agreed to review the PATTGEN doc and register spec.

Reply:
Great, thanks!